.github/workflows/ci-app-examples.yml

```diff
@@ -32,8 +32,8 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, macOS-11, windows-2022]
-        pkg-name: ["app"]  # TODO: add "LAI" # "LAI" installs the monolithic package
-        python-version: [3.8]
+        pkg-name: ["lightning", "app"]  # the "lightning" is for installing monolithic package
```

Comment:
I strongly suggest we add parametrizations separately, as I did in #15279

Reply:
You can revert the added parametrizations once CI is green here

Reply:
but why?

Reply:
Because I would like for us to really think about what jobs we will be adding, and adding them to checkgroup etc, but this PR is already pretty long, I don't want it to get huge.

Reply:
So are you suggesting making these tests run once and then dropping CI for its codebase and praying that all will still work even though we won't be testing it?
cc: @otaj @justusschock

Reply:
We often sequence pieces of work where a fix is pushed before the tests are added.

In fact, previous PRs of this line of work already merged worked without tests being ready.

I'm not saying that the changes shouldn't be tested, just that it would be a better experience for reviewers if we sequence it more finely.

Reply:
> We often sequence pieces of work where a fix is pushed before the tests are added.

really? that not good, that it is against the best practice

> In fact, previous PRs of this line of work already merged worked without tests being ready.

I don't think so; this CI was added before this PR, or are you referring to fixes in the package/test for PL, which was done on green status?

Reply:
pruned the jobs count here f1630e0786a04952bb9f1377645391165246aaba